dipy/nn/tests/test_cnn_1denoiser.py

```diff
@@ -26,12 +27,13 @@ def test_default_Cnn1DDenoiser_sequential():
 
 
 @pytest.mark.skipif(not have_tf, reason='Requires TensorFlow')
-def test_default_Cnn1DDenoiser_flow(pytestconfig):
+@set_random_number_generator()
+def test_default_Cnn1DDenoiser_flow(pytestconfig, rng):
```

Comment:
You will have to update `set_random_number_generator` to handle `pytestconfig` fixture parameter.
here is how you can update it:
```python
def set_random_number_generator(seed_v=1234):
    """Decorator to use a fixed value for the random generator seed.

    This will make the tests that use random functions reproducible.

    """
    def _set_random_number_generator(func):
        def _set_random_number_generator_wrapper(pytestconfig, *args, **kwargs):
            rng = np.random.default_rng(seed_v)
            kwargs['rng'] = rng
            signature = inspect.signature(func)
            if pytestconfig and 'pytestconfig' in signature.parameters.keys():
                output = func(pytestconfig, *args, **kwargs)
            else:
                output = func(*args, **kwargs)
            return output
        return _set_random_number_generator_wrapper
    return _set_random_number_generator
```

do not forget to `import inspect` at the top of `dipy/testing/decorator.py`

Let me know if you have any questions.